invenio_matcher/models.py

```diff
@@ -27,7 +27,8 @@ class MatchResult(object):
 
     """Matcher - represent a result."""
 
-    def __init__(self, record, score):
+    def __init__(self, id_, record, score):
+        self.id = id_
```

Comment:
Mh, can't we use the `record`'s own `recid` for this, instead of the one given by Elasticsearch? The API would look a little cleaner.